master/buildbot/buildslave/libvirt.py

```diff
@@ -195,7 +186,7 @@ def _find_existing_instance(self):
         domains = yield self.connection.all()
         for d in domains:
             name = yield d.name()
-            if name.startswith(self.name):
+            if name.startswith(self.slavename):
```

Comment:
I'm inclined toward `name`.  However, `.name` was not available.  It probably needs to be fixed up in the chain...